src/components/Inventory/index.tsx

```diff
@@ -146,11 +163,64 @@ export default function Inventory(props: InventoryProps): JSX.Element {
       facilityIds: filters.facilityIds,
       searchSortOrder,
     });
-    const updatedResult = apiSearchResults?.map((result) => {
-      const resultTyped = result as InventoryResult;
-      const facilityInventoriesNames = resultTyped.facilityInventories.map((nursery) => nursery.facility_name);
-      return { ...result, facilityInventories: facilityInventoriesNames.join('\r') };
-    });
+
+    let updatedResult: InventoryResultWithFacilityNames[] | undefined = [];
+    if (filters.facilityIds && filters.facilityIds.length) {
+      const nextResults: InventoryResultWithFacilityNames[] = [];
+      apiSearchResults?.forEach((result) => {
+        const resultTyped = result as FacilityInventoryResult;
+        const indexFound = nextResults.findIndex((res) => res.species_id === resultTyped.species_id);
+
+        if (indexFound !== undefined && indexFound !== -1) {
+          const existingSpecies = nextResults[indexFound];
+          nextResults[indexFound] = {
+            ...existingSpecies,
+            germinatingQuantity: (
+              Number(existingSpecies.germinatingQuantity) + Number(resultTyped['germinatingQuantity(raw)'])
+            ).toString(),
+            notReadyQuantity: (
+              Number(existingSpecies.notReadyQuantity) + Number(resultTyped['notReadyQuantity(raw)'])
+            ).toString(),
+            readyQuantity: (
+              Number(existingSpecies.readyQuantity) + Number(resultTyped['readyQuantity(raw)'])
+            ).toString(),
+            totalQuantity: (
+              Number(existingSpecies.totalQuantity) + Number(resultTyped['totalQuantity(raw)'])
+            ).toString(),
+            facilityInventories: `${existingSpecies.facilityInventories}, ${resultTyped.facility_name}`,
```

Comment:
Just curious, can a facility show up multiple times here?
Suggestion, keep this as an array (or Set) of facility names and do a `join(',')` in the final updatedResults.map.

Reply:
Each result comes by species and facilities, so if the species already exists, it's because that facility was already added, and we are in other combination of species-facility. So this would not happen, but I will create a set so it's clearer

Reply:
Ok, if it is guaranteed to be unique don’t worry. Should we sort the array?

Reply:
Np, let’s keep them the same and maybe revisit later.